README.md

```diff
@@ -1,2 +1,27 @@
 # openEQUELLA-slideshow
 Web component that leverages openEQUELLA content in a configurable slideshow
+
+The slideshow displays images from oEQ items.  To find the items to display, it leverages the search REST API of oEQ.
+
+The slideshow allows 'in app' configuration of the search query to dynamically change the slideshow. Alternatively, the slideshow can be configured with a `search-query` property the handles the configuration.
+
+For items with multiple images, the slideshow will iterate through all of them, displaying them one at a time.
```

Comment:
A good starting point, something which will likely come up would be having multiple items, paging, and autoplay

uPortal's slideshow components ended up adding these by popular request
https://uportal-contrib.github.io/uPortal-web-components/en/components/dashboard-carousel/demo
https://uportal-contrib.github.io/uPortal-web-components/en/components/content-carousel/demo

https://glidejs.com and https://kenwheeler.github.io/slick offer some good options for creating these types of experiences.

Reply:
I have a bit of experience with slick, zero with glidejs.  I like glidejs in that it seems we wouldn't be tied into a framework.  slick requires jquery.  not necessarily bad, but gives me a bit of pause.  Ideally, I'd like the main image to fill the screen with captions overlaid, and then have an optional carousel at the bottom to show prev / next.  Not sure if either of those support that, I'll have to do some research.

Reply:
It likely can, it's more what layer of abstraction would work best.
We could go higher level than glide or slick, with something like https://www.linxtion.com/demo/react-image-gallery, but it may be challenging to make it work with audio/video.
We could also go lower level, with something like https://www.react-spring.io/docs/hooks/use-springs or https://www.framer.com/api/motion/examples which give fine grained control/apis at the cost of needing to manage the animations, accessibility, and state.